packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -623,36 +624,41 @@ export default {
 
         /**
          * Check form is valid - no inline messages
-         * If form is valid try to call `submitCheckout`
-         * Catch and handle any errors
+         * 1. If form is valid try to call `submitCheckout`.
+         * 2. If the form is invalid process error tracking and logging via `onInvalidCheckoutForm()`.
          */
         async onFormSubmit () {
             this.trackFormInteraction({ action: 'submit' });
 
             if (!this.isFormValid()) {
-                const validationState = validations.getFormValidationState(this.$v);
-
-                this.$emit(EventNames.CheckoutValidationError, validationState);
-                this.trackFormInteraction({ action: 'inline_error', error: validationState.invalidFields });
-
-                this.$logger.logWarn(
-                    'Checkout Validation Error',
-                    this.$store,
-                    validationState
-                );
+                this.onInvalidCheckoutForm();
                 return;
             }
 
-            this.shouldDisableCheckoutButton = true;
+            await this.submitCheckout();
+        },
+
+        /**
+         * Fired when `isFormValid` returns truthy via `onFormSubmit()` call.
```

Comment:
Quick question , but why is the `onInvalidCheckoutForm` fired when `isFormValid` is truthy?

Reply:
my bad worded it wrong. 🤦

Reply:
Fixed.